source/Calamari.Azure/Calamari.Azure.csproj

```diff
@@ -0,0 +1,106 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>1.0.0.0</Version>
+    <TargetFramework>net452</TargetFramework>
```

Comment:
.net451 is part of [Microsoft Server 2012 r2](https://msdn.microsoft.com/en-us/library/8z6watww(v=vs.110).aspx#Supported server operating systems), I thought that was the minimum requirement correct ?

BTW, net451 is the minimum for OD.